### PR TITLE
shards: add page

### PR DESCRIPTION
**Pull request by aa-tan**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by aa-tan** [comment]
Debating whether to change "dependencies" to "shards" as that is what they're called by Crystal-lang.

File: pages/common/shards.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+`shards update`
+
+- List all installed dependencies:
+
+`shards list`
+
+- List version of dependency:
+
+`shards version {{path/to/dependency/folder}}`
```
Comment (pxgamer) [inline pages/common/shards.md]: Perhaps change this to `{{path/to/dependency_folder}}`

**Comment by pxgamer** [comment]
Sorry, this review was meant to be down as `Request changes` not `Accepted` 🤦‍♂️

**Comment by pxgamer** [comment]
Cheers @aa-tan 👌

File: pages/common/shards.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# shards
+
+> Dependency management tool for the Crystal language.
+
+- Create a skeleton shard.yml file:
+
+`shards init`
+
+- Install dependencies from shard.yml file:
```
Comment (sbrl) [inline pages/common/shards.md]: Perhaps ...`from a shard.yml`... would read better?

Comment (aa-tan) [reply pages/common/shards.md]: Made the change at 6ac0440 !

File: pages/common/shards.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- Update all dependencies:
+
+`shards update`
+
+- List all installed dependencies:
+
+`shards list`
+
+- List version of dependency:
```
[the remark this answers is not in the document]
Comment (aa-tan) [reply pages/common/shards.md]: `shards list` lists all installed dependencies and their versions as defined in shard.yml (This is run in the project folder)

Dependencies are stored within the project folder in a `lib` directory.
`shards version {{path/to/dependency_folder}}` lists the version of the dependency as defined by that dependency's shard.yml. Running `shards version` without supplying a path (assuming this is run in the project folder) will therefore give the version of the current project's shard.yml.
To clear any misunderstanding, I could add an example before this to indicate that. I could also make the change from `{{path/to/dependency_folder}}` to `{{lib/dependency_folder}}` if need be.

[the remark this answers is not in the document]
Comment (agnivade) [reply pages/common/shards.md]: Let us change this to `shards version` to make it simple.

**Pull request merged**